pennylane/ops/op_math/parametrized_evolution.py

```diff
@@ -156,6 +156,25 @@ class ParametrizedEvolution(Operation):
     >>> jax.grad(circuit)(params)
     Array([-4.8066125,  3.7038102, -1.3294725, -2.4061902,  0.6811545,
         -0.5226515], dtype=float32)
+
+    One can also provide a list of time values that the odeint will use to calculate the parametrized
+    hamiltonian's evolution:
```

Comment:
Maybe this is more a matter of understanding what the `odeint` does, so I don't know if more explanation really belongs here, but its not clear to me what this does that is different than `t=[0, 10]`. I can see the output is different, I just don't understand why.

Reply:
Oh the output should actually not be different!

When odeint does is it evaluates all `ts` from first to last and performs steps in between. So when you just pass `t=[0, 10]` its starting at 0 and moving onwards to 10 with adaptive step sizes. When we pass `t=linspace(0, 10, 0.001)` we are forcing `odeint` to evaluate all of these points (but in between it will still perform intermediate steps whenever the error estimate is not within the provided tolerance.) I briefly touch on this in my suggested change of the parameter `t` description above. I would add one similar sentence here to make this a bit clear.

Reply:
The outputs are different, though. I've confirmed running it in a notebook, changing nothing but `t=[0, 10]` vs `t=jnp.arange(0., 10., 0.1))`. What is included here is accurate.

Reply:
Updated example!!

Reply:
Ah thanks didnt realize this either!